### ISSUE TITLE
Option to Not Accumulate Increment Presenter Notes

### ISSUE THREAD

**Issue by jessebett** [gnab/remark#574]
Mentioned at the end of #561, the current way to incrementally show notes accumulates past notes. It would be nice to be able to turn this off.

```
a
???
notes for a
--
b
???
notes for b
```

but right now the presenter mode in `b` shows both `notes for a` and `notes for b`.

**Comment by igorsantos07** [comment]
yep, the accumulator is intended, as I understood, as a way to incrementally bring new content to the slides. as we're always discussing the new content on the screen, notes should never accumulate.

**Comment by gnab** [comment]
This has been fixed and will go into version 0.16.0.
